### ISSUE TITLE
Bug: Some senders cause the sidebar to error

### ISSUE THREAD

**Issue by heliostatic** [Foundry376/Mailspring#1921]
**Describe the bug**
Some senders cause an error in the sidebar:
![image](https://user-images.githubusercontent.com/70898/79742974-b84c3800-82d1-11ea-894d-beee5de15996.png)
```TypeError: Cannot read property 'startsWith' of null
    at SidebarParticipantProfile._renderPersonInfo (file:///var/folders/nz/vv4_9tw56nv9k3tkvyszvwg80000gn/T/nylas-build/electron-packager/darwin-x64/Mailspring-darwin-x64/Electron.app/Contents/Resources/app/internal_packages/participant-profile/lib/sidebar-participant-profile.tsx:491:48)
    at SidebarParticipantProfile.render (file:///var/folders/nz/vv4_9tw56nv9k3tkvyszvwg80000gn/T/nylas-build/electron-packager/darwin-x64/Mailspring-darwin-x64/Electron.app/Contents/Resources/app/internal_packages/participant-profile/lib/sidebar-participant-profile.tsx:513:15)
    at ah (/Applications/Mailspring.app/Contents/Resources/app.asar/node_modules/react-dom/cjs/react-dom.production.min.js:163:192)
    at $g (/Applications/Mailspring.app/Contents/Resources/app.asar/node_modules/react-dom/cjs/react-dom.production.min.js:162:264)
    at eh (/Applications/Mailspring.app/Contents/Resources/app.asar/node_modules/react-dom/cjs/react-dom.production.min.js:171:13)
```

**To Reproduce**
It looks like it is related to senders not having a name, maybe?

**Expected behavior**
Load the sender information even when there is no name.

**Screenshots**
<img width="285" alt="CleanShot 2020-04-19 at 07 02 30@2x" src="https://user-images.githubusercontent.com/70898/79743100-fa757980-82d1-11ea-95b0-915f0557367a.png">
<img width="294" alt="CleanShot 2020-04-20 at 06 37 28@2x" src="https://user-images.githubusercontent.com/70898/79743102-fb0e1000-82d1-11ea-969e-0ee39ac5edb8.png">

**Setup: (please complete the following information):**
 - OS: macOS
 - OS Version: 10.14.6
 - Mailspring Version: 1.7.5

**Additional context** 
Add any other context about the problem here. If this issue is related to a particular account or email service, be sure to mention it so this report is actionable!

**Comment by heliostatic** [comment]
Looks like it's actually coming from here: https://github.com/Foundry376/Mailspring/blob/4133537ddc3d9a3f2df6f6d8b382c2722253cd83/app/internal_packages/participant-profile/lib/sidebar-participant-profile.tsx#L493

**Comment by heliostatic** [comment]
Ah, looks like you already fixed it in https://github.com/Foundry376/Mailspring/commit/4133537ddc3d9a3f2df6f6d8b382c2722253cd83#diff-16a28c47cbd05b64b52037b0b9fdbdfc

**Comment by bengotow** [comment]
Hey folks! This will be resolved in the 1.7.6 release very soon - sorry for the hassle. I also fixed the `linkedin.handle = null` scenario on the backend, but many folks had data already cached in the client that was not re-fetched.

**Comment by bengotow** [comment]
Merging in to #1909!
